vanguard.py

```diff
@@ -0,0 +1,49 @@
+import sys
+from pymarc import Record, Field, MARCReader, MARCWriter
+
+#SILSLA-13
+
+def modify_5xx(record):
+	for old_field in range(500, 600):
+		if old_field != 590:
+			for fld in record.get_fields(old_field):
+				for sfld in fld.get_subfields('5'):
+					if sfld.startswith('CLU'):
+						fld.tag = '590'
+						record.add_ordered_field(fld)
+	return record
```

Comment:
Since the calling code is not using the return value, and we're passing a mutable object (`record`) by reference, changes made to `record` within the function are automatically available to the caller.  So these `return` statements can be removed.